.github/workflows/sync_native_dependency.yml

```diff
@@ -0,0 +1,27 @@
+name: Compare Dependency Constraints
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  compare_dependency_constraints_script:
+    runs-on: ubuntu-latest
+    if: github.repository == 'demisto/dockerFiles'
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.11"
+      - name: Install dependencies with pipenv
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv
+          pipenv install
+      - name: Run compare_dependency_constraints Script.
+        run: |
+          ls
+          echo "==== $(date): Starting environment setup... ===="
```

Comment:
```suggestion
```
we don't really setup much here, maybe in the `Install dependencies` step this would make more sense, but removing it would make just as much sense